core/src/main/scala/com/raphtory/algorithms/generic/filters/VertexQuantileFilter.scala

```diff
@@ -0,0 +1,136 @@
+package com.raphtory.algorithms.generic.filters
+
+import com.raphtory.algorithms.api.Bounded
+import com.raphtory.algorithms.api.GraphPerspective
+import com.raphtory.algorithms.api.Identity
+import com.raphtory.algorithms.generic.NodeList
+
+import scala.collection.Searching.Found
+import scala.collection.Searching.InsertionPoint
+import scala.language.implicitConversions
+import scala.math.Numeric.Implicits.infixNumericOps
+import scala.reflect.ClassTag
+
+/**
+  * {s}`EdqeQuantileFilter()`
+  * : Filtered view of the graph based on edge weight
+  *
+  *  This creates a filtered view of the graph where nodes are removed based on where their value of a given property/state lies in the global
+  *  distribution of vertex states, understood in this algorithm as percentiles. For example, one can create a view of the
```

Comment:
should be "quantiles" instead of "percentiles" as it is between 0 and 1 and not 0 and 100?